### PR TITLE
Add YouTube Music

### LINKED ISSUE #3090: YoutTube Music
Name: YouTube Music
Website: https://music.youtube.com/
Resources: SVG image: https://commons.wikimedia.org/wiki/File:Youtube_Music_logo.svg

### PR DESCRIPTION
**Pull request by thrash4life**
youtubemusic

**Issue:** Closes #3090

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

**[Build passed](https://travis-ci.com/github/simple-icons/simple-icons/builds/167566781)**

### CODE REVIEW

**Comment by thrash4life** [comment]
Hi! 👋! I have an iHeartRadio icon ready to go too. I'm too lazy for creating multiple forks, so I'll wait in the queue. :-)

**Comment by ericcornelissen** [comment]
> Hi! 👋! I have an iHeartRadio icon ready to go too. I'm too lazy for creating multiple forks, so I'll wait in the queue. :-)

Notice that [the commit for iHeartRadio](e755383e1d5bad2d64f1bfb176278875c5366bd1) has replaced YouTube Music in this Pull Request. Could you undo that (for example by force pushing 7cb2c9c815ab40190da49cb4a9f65d4bb3a1f458.

For the future, you don't have to create multiple forks. You simply have to create branches. To get started with that I recommend reading [this article](https://guides.github.com/introduction/flow/). If you want to create a branch now I recommend you first checkout to f2ac002a12eb1565f167b54a624910d31031cf0c and then create a branch for iHeartRadio. If you're using a command line you can use the sequence of commands below. If you need any help feel free to ask us here!

```
git checkout f2ac002a12eb1565f167b54a624910d31031cf0c
git checkout -b original-develop
git checkout -b icon-iheartradio
git commit -m "Add iHeartRadio"
git push origin icon-iheartradio
```

And to fix this Pull Request you can use

```
git checkout develop
git checkout -b tmp
git checkout 7cb2c9c815ab40190da49cb4a9f65d4bb3a1f458
git branch -D develop
git checkout -b develop
git push origin develop --force
```

**Comment by thrash4life** [comment]
🤦‍♂️ - sorry about that. Late night unintentional autopilot commit. I didn't know I could branch for this project as well. Will do and apologies for the inconvenience!

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6520,6 +6520,11 @@
             "hex": "FF0000",
             "source": "https://gaming.youtube.com/"
         },
+        {
+            "title": "YouTube Music",
+            "hex": "FF0000",
+            "source": "https://music.youtube.com/"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:6526]:
```suggestion
            "source": "https://partnermarketinghub.withgoogle.com/#/brands/"
```

**Comment by ericcornelissen** [comment]
Thanks a bunch for the contribution @thrash4life, it's much appreciated 🎉

**Pull request merged**